executor/revoke.go

```diff
@@ -309,7 +309,7 @@ func (e *RevokeExec) revokeColumnPriv(internalSession sessionctx.Context, priv *
 		if err != nil {
 			return err
 		}
-		sqlexec.MustFormatSQL(sql, " WHERE User=%? AND Host=%? AND DB=%? AND Table_name=%? AND Column_name=%?", user, host, dbName, tbl.Meta().Name.O, col.Name.O)
+		sqlexec.MustFormatSQL(sql, " WHERE User=%? AND Host=%? AND DB=%? AND lower(Table_name)=%? AND Column_name=%?", user, host, dbName, strings.ToLower(tbl.Meta().Name.O), col.Name.O)
```

Comment:
```suggestion
		sqlexec.MustFormatSQL(sql, " WHERE User=%? AND Host=%? AND DB=%? AND lower(Table_name)=%? AND Column_name=%?", user, host, dbName, tbl.Meta().Name.L, col.Name.O)
```

For the type `CIStr`, you can use `CIStr.L`

Reply:
THKS MUCH!!!